### ISSUE TITLE
Both (Linux) desktop and (Android) phone apps unresponsive

### ISSUE THREAD

**Issue by zonotope** [keybase/client#13871]
Hi. My team uses keybase as our main group chat application among other things. Everything seemed to be working fine for me until yesterday. It looked like two messages came in, but they were never fully downloaded or decrypted, so I just got an orange dot next to the team chatroom to signify that new messages were waiting, and a loading spinner in the room where the messages would normally be. I could still see all the old messages, but the new ones never came through. I tried to send a message to the room, and there was a lock animation that never stopped on the message I tried to send, and the others on my team said they never got it. When I restarted the entire keybase daemon and app, it just hung on the splash screen and says "Loading". 

When I tried to connect with the keybase app on my phone, it also just hangs on the splash screen until Android reports that the app is not responding and asks me to kill it. 

When I tried `keybase log send` from the terminal, I got these error messages:
```
▶ INFO ignoring error getting keybase status: EOF
▶ INFO ignoring UI logs: dial unix /run/user/1000/keybase/keybased.sock: connect: connection refused
▶ ERROR API network error: Post https://api-0.core.keybaseapi.com/_/api/1.0/logdump/send.json: net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)
```

I am still logged in to the website, but I can't use keybase at all either from my computer or my phone, and it doesn't seem like I can send you the logs either. Everything seems to work fine for the other members of my team, so it looks like it's just my account, but it's weird that I'm seeing basically the same issue both on desktop and mobile.

How do I fix this?

**Comment by maxtaco** [comment]
Can you send feedback from your Android phone?

**Comment by maxtaco** [comment]
Are you behind a firewall and/or do you have network issues that just started? That would explain everything you described, though probably it would have affected other software too.

**Comment by zonotope** [comment]
I'm not behind a firewall (that I know of at least) and I can connect to
other websites and use other apps fine. I've only noticed issues with
keybase, and those issues just started yesterday afternoon (I've been using
keybase for a while on different teams without any problems before
yesterday). I can't think of anything that changed about my network
yesterday afternoon, but I just turned off wifi on my phone and it seems to
have worked. I just sent feedback that way if it's helpful. I can't think
of what could be blocking connections to keybase but letting everything
else through.

**Comment by maxtaco** [comment]
Yeah your isp seems to be blocking our host, from what you’ve said....
can you check if dns resolves for that domain? Or if you can open a tcp
connection to that host on port 443?
